### DISCUSSION TITLE
Volte vowifi toggle on, but volte not working in Indonesia, carrier Telkomsel

### DISCUSSION THREAD

**Discussion by Bloodiez** [kyujin-cho/pixel-volte-patch#108] [Carrier-specific Experiences]
I have follow the instructions, and the toggle for volte and vofiwi in switch on, but the volte sign not showing up(not working, even in the volte area),

IMS status both sim : not registered
Location : indonesia
Carrier : telkomsel
Device : oneplus 7 pro GM1913
Os : oos 12 , latest stable (h36)

Thanks

**Comment by Bloodiez** [comment]
Fix :
Oos 12.1 no root
If you still have problem (ims not registered, or not getting volte after toggle on the switch).

Dial
*#9434#
Copy imei and stamp
Find token authentication service(pay to get the token)
Unlock engineering mode (by input the token)

Dial
*#649010#
Unlock full port (code 6776)

Download qpst with pdc
(Make sure you have proper driver for your phone)
Install it
Find pdc apps in the qpst folder
Deactivate active profile,
Find the profile for sim 1(sub 0) and sim2 (sub1) and Activate it , then wait until you see volte icon appear in your phone, or try another profile.

Tried in my device oneplus 7pro
Profile india vodafone work in Indonesia region.

**Reply by Alexcereceda** [reply]
> Fix : Oos 12.1 no root If you still have problem (ims not registered, or not getting volte after toggle on the switch).
> 
> Dial *#9434# Copy imei and stamp Find token authentication service(pay to get the token) Unlock engineering mode (by input the token)
> 
> Dial *#649010# Unlock full port (code 6776)
> 
> Download qpst with pdc (Make sure you have proper driver for your phone) Install it Find pdc apps in the qpst folder Deactivate active profile, Find the profile for sim 1(sub 0) and sim2 (sub1) and Activate it , then wait until you see volte icon appear in your phone, or try another profile.
> 
> Tried in my device oneplus 7pro Profile india vodafone work in Indonesia region.

Where can i pay for the token? This generator https://one.andro.plus/# doesnt work for me.

**Reply by Bloodiez** [reply]
There's a lot of service providing you the token, you may find it in xda forum.

**Reply by Bloodiez** [reply]
Don't delete the default profile, just select the preferable profile and done. Should be still working even after reboot.

**Reply by Elgeweily** [reply]
I have the same issue with OP7T Pro, OOS12, No root.....followed your steps exactly, but when I open PDC, the dropdown list is empty, my device is not showing there, any suggestions please?

**Comment by Bloodiez** [comment]
Resolved
